dtfs-central-api/src/services/state-machines/fee-record/event-handlers/helpers/update-facility-utilisation-data.test.ts

```diff
@@ -42,17 +33,17 @@ describe('updateFacilityUtilisationData', () => {
       start: { month: 6, year: 2026 },
       end: { month: 7, year: 2027 },
     };
-    const utilisation = 9876543.21;
     const ukefShareOfUtilisation = 1234567.77;
     const requestSource: DbRequestSource = {
       platform: REQUEST_PLATFORM_TYPE.TFM,
       userId: 'abc123',
     };
+    const fixedFee = 12345;
```

Comment:
```suggestion
    const mockFixedFee = 12345;
```

Reply:
Actually it looks like test's mocks do not have "mock" prefixes - would prefer to be aligned.